src/api.js

```diff
@@ -112,8 +112,34 @@ async function inject(filename, resourceName, resourceData, options) {
     throw new Error("Error when injecting resource");
   }
 
+  const SENTINEL = "POSTJECT_SENTINEL_fce680ab2cc467b6e072b8b5df1996b2:";
+  const buffer = Buffer.from(data.buffer);
+  const firstSentinel = buffer.indexOf(SENTINEL);
+
+  if (firstSentinel === -1) {
+    throw new Error(`Could not find the sentinel ${SENTINEL} in the binary`);
```

Comment:
CI is failing because this error is getting thrown. It happens because the Node.js binary we are testing against doesn't do `#include "postject-api.h"` which is supposed to embed the sentinel string.

Suggestions on what we should do here?

Reply:
Postject is supposed to be completely generic and independent from i.e. Node.js or a specific application. Can we run the tests not against Node.js specifically, but mainly against a dummy program we compile from source during the testing phase?

Reply:
@jviotti Are you suggesting that we should remove this whole thing - https://github.com/postmanlabs/postject/blob/30e9a23044f71ca16f9345e94266906acb931d37/test/cli.mjs#L150-L233?

Reply:
Yeah, I think it makes sense. It feels that such test should be part of Node.js or our SEA CLI instead?

Reply:
Cool, rebased. Tests pass locally now but we would need to reconfigure CircleCI to get a green CI. PTAL.